scripts/docker_push.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+APP_NAME = data-sync-server
```

Comment:
Bash sadly does not allow spaces in variable declarations :)

It should be like this

```
APP_NAME=data-sync-server
```